fybrik/README.md

```diff
@@ -15,12 +15,11 @@ You will need a copy of the Fybrik repository (`git clone https://github.com/fyb
 
 1. Install Fybrik Prerequisites. Follow the instruction in the Fybrik [Quick Start Guide](https://fybrik.io/dev/get-started/quickstart/). Stop before the "Install control plane" section.
 
-1. Before installing the control plane, we need to customize the [Fybrik taxonomy](https://fybrik.io/dev/tasks/custom-taxonomy/), to define new connection and interface types. This customization requires [go](https://go.dev/dl/) version 1.17 or above. Please note that in this tutorial a built-in catalog called `katalog` is used as opposed to the default Openmetadata catalog used in Fybrik. Run:
+1. Install Fybrik with built-in catalog called `katalog`. This is used as opposed to the default Openmetadata catalog used in Fybrik.
```

Comment:
Install Fybrik with a built-in catalog, called Katalog, as opposed to the Openmetadata catalog which is installed by default.

Reply:
Fixed. Thanks